portfolio/src/main/webapp/script.js

```diff
@@ -46,7 +46,35 @@ function rotateImages() {
   setTimeout("rotateImages()", time);
 }
 
+function showHideCommentForm() {
+  fetch('/login-status')
+    .then(response => response.json())
+    .then((json) => {
+      if (json['loggedIn'].localeCompare("true")) {
+        loginURL = json['loginUrl'];
+        document.getElementById("comment-form").innerHTML = "<p>If you would like to comment" +
+          " please sign in using the link below: <br><br>" +
+          "<a href=\"" + loginURL + "\">Login</a></p>";
```

Comment:
Nit: instead of assembling Strings with arithmetic, it may be simpler to use template literals:

```
`<a href="${loginUrl}">Login</a>`
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals